### DISCUSSION TITLE
[Errno 2] No such file or directory: '<path>/node_modules/.bin/mjml

### DISCUSSION THREAD

**Discussion by kev26** [mjmlio/mjml#2496] [Q&A] [answered]
I'm new learner and using mjml, and when I trying to install mjml for my project, it appear this error:
```
[Errno 2] No such file or directory: '/home/kev/work/project/frontend/node_modules/.bin/mjml'
Check that mjml is installed and allow permissions for execute.
See https://github.com/mjmlio/mjml#installation
```
![Screenshot from 2022-07-02 10-27-24](https://user-images.githubusercontent.com/25315610/176985008-9f6d66b8-9712-4e7c-95eb-e07de44e6d9e.png)

**Comment by iRyusa** [comment, marked as answer]
Looks like you have no `node_modules` folder here  ?
